README.md

```diff
@@ -1 +1,91 @@
-pretend this readme is as interesting as the website
+![Logo](https://pbs.twimg.com/profile_images/1373353833330475008/rKWjiXET_400x400.png)
+
+
+# Bit Project Website
+
+This GitHub repo is the home 🏠 of the Bit Project website. 
+Here at the Bit Project We equip students with technical 
+superpowers by providing cutting-edge, open source learning 
+experiences to prepare students for careers in tech. 
+
+
+## Bit Project Sponsors
+
+ - ⭐️ [Microsoft](https://www.microsoft.com/en-us/)
+ - ⭐️ [Courier](https://www.courier.com/)
+
+
+
+## Projects From Former Bit Project Students
+
+Too see exactly the kind of work we do at Bit Project, 
+feel free to check out some student projects from 
+
+- Grace built [EcoMaps](https://www.bitproject.org/projects/reducing-carbon-emissions) and reduced carbon emissions 🌱
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://www.github.com/octokatherine)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://www.github.com/octokatherine)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://www.github.com/octokatherine)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://www.github.com/octokatherine)
+- 💙 Misha Patel, Director of Partnerships [Github Link](https://www.github.com/octokatherine)
+- 💜 Beatrix Cendana, Student Community Manager [Github Link](https://www.github.com/octokatherine)
+
+
+
+## API Reference
+
+#### Get all items
+
+```http
+  GET /api/items
+```
+
+| Parameter | Type     | Description                |
+| :-------- | :------- | :------------------------- |
+| `api_key` | `string` | **Required**. Your API key |
+
+#### Get item
+
+```http
+  GET /api/items/${id}
+```
+
+| Parameter | Type     | Description                       |
+| :-------- | :------- | :-------------------------------- |
+| `id`      | `string` | **Required**. Id of item to fetch |
+
+#### add(num1, num2)
+
+Takes two numbers and returns the sum.
+
+## Media Toolkit - Color Codes
+
+| Color             | Hex                                                                |
+| ----------------- | ------------------------------------------------------------------ |
+| Example Color | ![#0a192f](https://via.placeholder.com/10/0a192f?text=+) #0a192f |
+| Example Color | ![#f8f8f8](https://via.placeholder.com/10/f8f8f8?text=+) #f8f8f8 |
+| Example Color | ![#00b48a](https://via.placeholder.com/10/00b48a?text=+) #00b48a |
+| Example Color | ![#00d1a0](https://via.placeholder.com/10/00b48a?text=+) #00d1a0 |
+
+
+## FAQ
+
+#### Question 1
+
+Answer 1
+
+#### Question 2
+
+Answer 2
+
+
+## Feedback
```

Comment:
FAQ: 

**Are your programs free?**
Yes! Both our mentorship program and global student community are free for all members to join. However, the mentorship program is highly selective and requires an application.

**How can I contribute?**
Email us at info@bitproject.org and we'll be happy to work with you!